Containerfile

```diff
@@ -15,6 +15,8 @@ RUN dnf install -y \
     $(python3 setupcfg2rpm.py setup.cfg) \
     && dnf clean all
 
+RUN pip install --upgrade sentry-sdk[flask]==0.14.2
```

Comment:
I'd unpin it (see https://github.com/packit/packit-service/pull/738) (latest is `0.16.5`)

Reply:
thanks, done